src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -224,7 +224,7 @@ static <T extends LfElement> void updateRemoteVoltageControlEquations(VoltageCon
                         .setActive(false);
                 equationSystem.getEquation(controllerElement.getNum(), ctrlEqType)
                         .orElseThrow()
-                        .setActive(false);
+                        .setActive(!controllerElement.isDisabled());
```

Comment:
There is something, I don't understand: vl.isDisabled() check that controlled bus and ALL controller buses are disabled, so this should be always false and equivalent to actual code.

Reply:
Understood, check should not be done on voltage control but just the controlled bus.
I pushed a fix.